servant-foreign/servant-foreign.cabal

```diff
@@ -0,0 +1,27 @@
+name:                servant-foreign
+version:             0.5.0.0
```

Comment:
Our `bump-version` script forces everything to be consistent, even to extra 0s. As does our upload script, I believe. So yeah, 0.5 is better, but it'll be fixed sooner or later.